roles/ngi_reports/tasks/main.yml

```diff
@@ -9,10 +9,9 @@
        recursive=yes
 
 - name: Install ngi_reports
-  shell: "{{ sw_path }}/anaconda/envs/{{ NGI_venv_py3_name }}/bin/pip install -e ."
-  args:
-    chdir: "{{ ngi_reports_dest }}"
-
+  pip:
```

Comment:
I'm not sure if this gets installed the right way. I tried running it but got an error. Maybe `executable` works better instead of `virtualenv`?

Reply:
Good catch!
The fix is embarrassingly simple and took entirely too long to find :grimacing: